updates existing yuba data and runs .Rmd, generating data objects

liz-fw opened the pull request:
closes #40

merging into `combined-habitat-updates`

MaddeeRubenson commented on data-raw/watershed/yuba/yuba_river.rmd:
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Yuba River"
 author: "[Sadie Gill](mailto:sgill@flowwest.com), [Mark Tompkins](mailto:mtompkins@flowwest.com)"
```
Per the comment on mine, should we remove Sadie here?

liz-fw replied:
yes, i'll remove! i'll also put the suitability criteria in table format to be more aligned with your markdown

MaddeeRubenson commented on data-raw/watershed/yuba/yuba_river.rmd:
```diff
@@ -15,17 +15,85 @@ library(scales)
 **Data Source:** [Spawning FWS 2010](https://www.fws.gov/lodi/instream-flow/Documents/Yuba%20River%20Spawning%20Final%20Report.pdf){target="_blank"} (pgs 59-61) and  [Rearing FWS 2010](https://www.fws.gov/lodi/instream-flow/Documents/Yuba%20River%20Rearing%20Final%20Report.pdf){target="_blank"} (pgs 74-76)
 
-The instream spawning and rearing habitat for Fall Run Chinook Salmon, Spring Run Chinook Salmon, and Steelhead in the Yuba River are based on data from the Fish and Wildlife Service (FWS) flow-habitat relationship final reports. A 2-dimensional hydraulic and habitat model called River2D was used to model available spawning and rearing habitat. The FWS instream spawning and rearing data were prov ...
+The instream spawning and rearing habitat for Steelhead in the Yuba River are based on data from the Fish and Wildlife Service (FWS) flow-habitat relationship final reports. A 2-dimensional hydraulic and habitat model called River2D was used to model available spawning and rearing habitat. The FWS instream spawning and rearing data were provided by [Mark Gard](mailto:Mark.Gard@Wildlife.ca.gov) fr ...
```
This is a bit pedantic but I feel like we should be consistent between "instream", "in channel", and "in-channel". Not sure which is best but being consistent through the markdown is probably good.

MaddeeRubenson commented on data-raw/watershed/yuba/yuba_river.rmd:
```diff
@@ -15,17 +15,85 @@ library(scales)
+
+chinook <- left_join(chinook_ic_rear, chinook_spawn) |> 
+  # TODO confirm this assumption that two reaches are the same
+  mutate(reach = ifelse(reach == "below_dpd", "dag_fea", "eng_dag")) |> 
```
agreed that this seems like an important thing to double check

MaddeeRubenson commented on data-raw/watershed/yuba/yuba_river.rmd:
```diff
@@ -150,26 +222,55 @@ source('data-raw/watershed/floodplain_utils.R')
+#   arrange(flow_cfs)
 
-threshold <- yuba_fp$flow_cfs[min(which(yuba_fp$FR_floodplain_acres > 0) - 1)]
+threshold <- yuba_river_floodplain$flow_cfs[min(which(yuba_river_floodplain$FR_floodplain_acres > 0) - 1)]
```
this doesn't seem necessary because floodplain acres doesn't start until 1500 regardless

MaddeeRubenson commented on data-raw/watershed/yuba/yuba_river.rmd:
````diff
@@ -150,26 +222,55 @@ source('data-raw/watershed/floodplain_utils.R')
+```{r, echo = FALSE, warning = FALSE, message = FALSE}
+# read in updated data
+# assign spring run and steelhead to fall run
+yuba_river_floodplain <- read_excel("data-raw/watershed/yuba/data/HydraulicHabFlow_LYR_7-15-2024.xlsx",
````
ugh - I just don't know if this should be considered a WUA. I think we need more information on the reaches - would it make sense to do something additive like this? However, the total inundated acres is still much lower than the previous Yuba floodplain acres. I think this requires more discussion. 

read_excel("data-raw/watershed/yuba/data/HydraulicHabFlow_LYR_7-15-2024.xlsx",
                              sheet = "LYR_HabFlowRelationships",
                              range = "A4:G40") |> 
    janitor::clean_names() |> 
  select(flow_cfs, below_dpd = below_dpd_6,
         above_dpd = above_dpd_7) |> 
  mutate(FR_floodplain_acres = below_dpd + above_dpd) |> 
  select(-below_dpd, -above_dpd) |> 
  ggplot() +
  geom_line(aes(flow_cfs, FR_floodplain_acres))

liz-fw replied:
noting here that I will follow up with Yuba team to confirm that their floodplain values (and spawning and rearing) are lower than previous existing values, and will check with mark.

liz-fw commented:
Hi Maddee, we can find your comments when you're back but I wanted to take some notes here based on what you said in slack -- 

- floodplain: they provided us floodplain acres for two different reaches. is there a way to combine them without converting to wua (i.e., using the lengths of the river miles to combine them, which is what I did?)
- removed the floodplain threshold
- fixed nomenclature for in-channel vs instream
